client/src/pages/people/Show.tsx

```diff
@@ -357,14 +367,13 @@ const PersonShow = ({ pageDispatchers }: PersonShowProps) => {
                     label="Attachments"
                     component={FieldHelper.ReadonlyField}
                     humanValue={
-                      <div className="attachment-card-list">
-                        {person.attachments.map(attachment => (
-                          <AttachmentCard
-                            key={attachment.uuid}
-                            attachment={attachment}
-                          />
-                        ))}
-                      </div>
+                      <AttachmentsDetailView
+                        attachments={attachments}
+                        updateAttachments={updateAttachments}
+                        relatedObjectType={Person.relatedObjectType}
+                        relatedObjectUuid={person.uuid}
+                        allowEdit={canEdit}
```

Comment:
This is not entirely correct; whether attachment edit is enabled also depends on `Settings.fields.attachment.restrictToAdmins` (see **people/Form.tsx**).